kms/azurekms/key_vault_test.go

```diff
@@ -72,6 +72,22 @@ func Test_now(t *testing.T) {
 	}
 }
 
+func TestRegister(t *testing.T) {
+	fn, ok := apiv1.LoadKeyManagerNewFunc(apiv1.AzureKMS)
+	if !ok {
+		t.Fatal("azurekms is not registered")
+	}
+	k, err := fn(context.Background(), apiv1.Options{
+		Type: "azurekms", URI: "azurekms:",
+	})
+	if err != nil {
+		t.Fatalf("New() error = %v", err)
+	}
+	if k == nil {
+		t.Fatalf("New() = %v, want &KeyVault{}", k)
+	}
```

Comment:
You could use the `require.*` functions for these assertions to make it shorter. Not a blocker.

Reply:
I'm not used to using testify on crypto before, in fact, there is only one file using it.

Reply:
Al right 🙂 

Unless there's a strong objection, soon there'll be more test using those packages.